src/components/VideoDetail/RelatedVideos/RelatedVideos.component.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { RelatedVideosItem } from './RelatedVideosItem/RelatedVideosItem.component';
+import styled from 'styled-components';
+import { useFetchRelatedVideos } from '../../../utils/hooks/useFetchRelatedVideos';
+
+const Container = styled.div`
+    padding 0px 20px 0px 20px;
+
+    a{
+        color:black;
+    }
+`;
+
+export const RelatedVideos = ({videoId}) => {
+
+    const { videos} = useFetchRelatedVideos(videoId);
+
+    return <Container>
+        {   
+            videos!==null && videos.items.length>0 &&
+            videos.items.map(video =>                
```

Comment:
When you have multiple conditions you can create a variable with an expressive name to know what's happening, for example:

```jsx
const hasVideos = videos !== null && videos.items.length > 0;
```
and when used in the mark up it's almost like natural language

```jsx
return (
  <Container>
    {hasVideos && videos.items.map(...)}
  </Container>
);
```